src/engraving/dom/engravingitem.cpp

```diff
@@ -1270,6 +1277,77 @@ void EngravingItem::manageExclusionFromParts(bool exclude)
     }
 }
 
+bool EngravingItem::appliesToAllVoicesInInstrument() const
+{
+    return hasVoiceApplicationProperties()
+           && getProperty(Pid::APPLY_TO_VOICE).value<VoiceApplication>() == VoiceApplication::ALL_VOICE_IN_INSTRUMENT;
+}
+
+void EngravingItem::setInitialTrackAndVoiceApplication(track_idx_t track)
+{
+    if (track == muse::nidx) {
+        track = m_track;
+    }
+    if (MScore::dynamicsApplyToAllVoices) {
```

Comment:
I'm not a big fan of these global flags... Can we add it to the engraving configuration instead?